indico/modules/events/timetable/blueprint.py

```diff
@@ -53,5 +54,7 @@
                  'add_session_block', RHLegacyTimetableAddSessionBlock, methods=('GET', 'POST'))
 _bp.add_url_rule('/manage/timetable/add-session', 'add_session', RHLegacyTimetableAddSession, methods=('GET', 'POST'))
 
+
 # Display
 _bp.add_url_rule('/timetable/', 'timetable', RHTimetable)
+_bp.add_url_rule('/timetable/timetable.pdf', 'export_pdf', RHLegacyTimetableExportPDF, methods=('GET', 'POST'))
```

Comment:
I'd go for `/timetable/pdf`. If it was just a simple PDF download without any settings I'd go for `timetable.pdf` too but since you show a form first I don't think a URL with .pdf is a good idea